source/main/physics/BeamFactory.cpp

```diff
@@ -886,6 +886,7 @@ void BeamFactory::calcPhysics(float dt)
 		m_trucks[t]->handleResetRequests(dt);
 		m_trucks[t]->handleTruckPosition(dt);
 		m_trucks[t]->updateAngelScriptEvents(dt);
+		m_trucks[t]->UpdatePropAnimations(dt);

```

Comment:
This will call `calcAnimators()` regardless of the `state` of the truck. Do we want that?

[So far](https://github.com/mikadou/rigs-of-rods/blob/0f165f036be41b8fbcceb10ee046653ffac5e829/source/main/physics/BeamForcesEuler.cpp#L1160) it was only called for trucks with states below `SLEEPING`.